### DISCUSSION TITLE
Iniciante em python

### DISCUSSION THREAD

**Discussion by Noc1vo** [frontendbr/forum#2046] [Perguntas e Respostas]
Olá pessoal, estou tentando aprender programação e queria focar na  área de DataScience já estudo estatística e é basicamente aliar isso a programação. Queria saber onde posso encontrar material de forma didática sobre esta área, de preferencia em português, meu inglês é só o básico. Mas, pode sugerir em inglês também.

**Comment by lddv** [comment]
Indico o canal [Programação Dinâmica](https://www.youtube.com/channel/UC70mr11REaCqgKke7DPJoLg). Estou tentando estudar um pouco para orientar a minha irmã nessa área e estou usando os vídeos deles como referência.

Para aprender lógica de programação, recomendo o [curso de Python do Guanabara](https://youtu.be/S9uPNppGsGo).

Ambos em português, espero que ajude.

**Comment by sbrj** [comment]
Para anáise de dados, tem um curso online do [LabHAcker](https://www.youtube.com/watch?v=W_Bz7M91R1Q) que usa python, pandas, matplotlib e jupyter notebook. Concordo com o amigo acima que falou sobre o canal Programação dinâmica e o curso do Guanabara. Vale ressaltar que o Guanabara tb fornece um cuso apenas de lógica de programação, para assistir antes de começar o de Python. Ele tem muita didática, então realmente vale a pena, pois sai entendendo realmente. Fazer todos os exercícios é fundamental tb.

**Comment by HeyCaroll** [comment]
Para inicio de estudo em Python recomendo o curso online do [curso em video](https://www.cursoemvideo.com/cursos/) ministrado pelo professor Gustavo
